api/src/test/java/org/pmiops/workbench/api/ConceptsControllerTest.java

```diff
@@ -241,7 +242,7 @@ public void setUp() {
     // SpringBootTest, which causes problems with CdrDbConfig. Just construct the service and
     // controller directly.
     ConceptService conceptService = new ConceptService(entityManager, conceptDao);
-    conceptsController = new ConceptsController(conceptService, workspaceService,
+    conceptsController = new ConceptsController(bigQueryService, conceptService, workspaceService,
```

Comment:
Do we need a test case in here that test this new endpoint? Or is that coming in a later PR?

Reply:
I think I am going to a followup PR with tests.

Reply:
I ended up having the time, so I just added a contract test.